DailyCodingProblem/Problems1Through25.cs

```diff
@@ -0,0 +1,94 @@
+﻿using System;
+using System.Collections.Generic;
+
+namespace DailyCodingProblem
+{
+    public class Problems1Through25
+    {
+        public static void UI_Problem1()
+        {
+			List<int> numbers = new List<int>() { 1, 2, 4, 5, 8, 9 };
+
+			while (true)
+			{
+				Console.WriteLine($"\nThe list is: {string.Join(", ", numbers)}");
+				Console.Write("Please enter a sum: ");
+
+				int sum;
+				while (!int.TryParse(Console.ReadLine(), out sum))
+				{
+					Console.Write("Error... Please enter a valid integer: ");
+				}
+
+				Problem1(numbers, sum);
+			}
+		}
+
+		/// <summary>
+		/// Given a list of numbers and a number k, return whether 
+		/// any two numbers from the list add up to k. 
+		/// 
+		/// For example, given[10, 15, 3, 7] and k of 17, 
+		/// return true since 10 + 7 is 17.
+		/// </summary>
+		protected static bool Problem1(List<int> numbers, int sum)
+		{
+			// Loop through the list trying every 2 element combination
+			for (int i = 0; i < numbers.Count - 1; i++)
+			{
+				for (int j = i + 1; j < numbers.Count; j++)
+				{
+					if (numbers[i] + numbers[j] == sum)
+					{
+						Console.WriteLine($"Yes, {numbers[i]} + {numbers[j]} = {sum}");
+						return true;
+					}
+				}
+			}
+
+			Console.WriteLine($"No two numbers in the listed added up to {sum}");
+			return false;
+		}
+
+		public static void UI_Problem2()
+		{
+			var numbers1 = new List<int>() { 1, 2, 3, 4, 5 };
```

Comment:
It would be nice if you asked the user for the array.

Reply:
I plan on deleting the UI functions now that I'm doing them as test driven